operations/finance/risk-management/mattermost-templated-agreements.md

```diff
@@ -6,18 +6,18 @@ description: >-
 
 # Mattermost U.S. Consulting Agreements
 
-This agreement is typically used for vendors who do not require Mattermost to sign a [custom agreement](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#procuring-a-vendor). Before you kick off the process, please confirm your vendor satisfies the criteria below. If you are not sure, please reach out to [People Ops](https://community.mattermost.com/private-core/channels/ask-people-team).
+This agreement is typically used for vendors who do not require Mattermost to sign a [custom agreement](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#procuring-a-vendor). Before you kick off the process, please confirm your vendor satisfies the criteria below. If your vendor does not satisfy the criteria below, this vendor contract should follow the [Procurement Process](https://handbook.mattermost.com/operations/finance/purchasing/how-to-procure-a-vendor-contract). If you are not sure, please reach out to [People Ops](https://community.mattermost.com/private-core/channels/ask-people-team).
 
 ## **Criteria:**
 
-* A firm or an individual with an LLC 
+* An individual with an LLC 
 * Provide ad-hoc services to Mattermost, examples include:  
-  * Project-Based: for a defined period with pre-set project milestones
-  * Ad-hoc services: work that is \(i\) not continuous \(e.g. hours vary month to month\) and \(ii\) typically assigned on an as-needed basis
-
+  * On-going nature: the engagement will exceed 6 months and does not have a fixed end date, e.g. on-going budget approval will not be needed. 
+  * System Access requirements: the engagement will require system access currently controlled by IT and/or HR. 
+  
 ## **Process**
 
-1. Confirm you have [budget approval](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#step-1-send-for-budget-approval) to engage with a vendor.
+1. Confirm you have budget approval for this with your Team MLT. If you do not have budget approval, see:  [off-cycle budget process](https://handbook.mattermost.com/operations/finance/budget#off-cycle-budget-approval).
 2. The appropriate [CAO](https://handbook.mattermost.com/operations/operations/company-agreements#what-are-e-sign-completion-expectations) should complete the [Mattermost U.S. Consulting Agreement Template](https://app.hellosign.com/s/GnvOJVYW). Reference the [field description guide](https://docs.google.com/document/d/1FKef8xf9N_NTEc0owbAGH4mb0UzU8CJQ5qckGvIl1UM/edit) for help. 
 3. Upon completion, the CAO will receive the document via HelloSign. Forward that email to [HR](mailto:%20hr@mattermost.com) and include your vendor's signatory contact information \(full name and signature\). 
```

Comment:
If these are vendors, they should run through Procurement and HR is no longer part of the process. Legal took over the consultant agreement process from HR from a vendor/procurement perspective.  However, we shouldn't be having anyone sign an agreement before it has gone through the procurement process, which includes PO approval. If they are agreeing to our standard template, then legal review is not necessary.